Update config docs (uwsgi >> gunicorn) (#13456)
galaxyproject/galaxy: doc/source/admin/config.rst

----------------------------

- Edit ``config/galaxy.yml`` (copy it from ``config/galaxy.yml.sample`` if it does not exist) to make configuration
  changes. This is a `YAML configuration file`_ and should contain one or two sections, one named ``gravity`` for

>   Gunicorn server options (optional) and one named ``galaxy`` for Galaxy (required).

Reviewer: ```suggestion
  process management via `Gravity`_ (optional) and one named ``galaxy`` for Galaxy (required).
```

>     - The default port for the Galaxy web server is ``8080``, and it only binds to localhost by default. To configure

Reviewer: ```suggestion
    - The `gravity` section contain several important options about how to
      launch Galaxy web workers and job handlers, which are documented in the
      :doc:`Scaling and Load Balancing <scaling>` documentation.
```

>       Gunicorn to listen on all available network addresses, set ``bind`` in the ``gravity`` section of ``galaxy.yml``
>       to ``0.0.0.0:<port>`` (e.g. ``bind: 0.0.0.0:8080``).
>     - Some Gunicorn options are required for Gunicorn to run Galaxy properly and will be added to the ``gunicorn``
>       command line by `Gravity` via the ``run.sh`` script if not specified in ``galaxy.yml``.
>     - Gunicorn has a `large number of options`_. The Galaxy documentation refers to some of them, but many additional
>       advanced deployment scenarios are available.

Reviewer: ```suggestion
```

Author: You mean we should provide these options and their documentation via Gravity?

Author: Oh I see, your previous edit. Never mind.

- Run Galaxy with ``sh run.sh``
- Use a web browser and go to the address you configured in ``galaxy.yml`` (defaults to http://localhost:8080/)

> .. _uWSGI YAML configuration file: https://uwsgi-docs.readthedocs.io/en/latest/Configuration.html

Reviewer: ```suggestion
```

> .. _large number of options: https://docs.gunicorn.org/en/latest/settings.html

Reviewer: ```suggestion
```
